data/network/src/commonMain/kotlin/social/androiddev/common/network/di/NetworkModule.kt

```diff
@@ -38,6 +38,14 @@ val networkModule: Module = module {
                     }
                 )
             }
+//            install(Auth) {
```

Comment:
👍🏼

Reply:
yeah I wasn't sure what to do with this, I usually did auth through okhttp and need to learn how ktor wants to handle it. For now was just passing manually from repository